test/parallel/test-webcrypto-derivebits-cfrg.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const { webcrypto } = require('crypto');
+const { subtle } = webcrypto;
+
+const kTests = [
+  {
+    name: 'X25519',
+    size: 32,
+    pkcs8: '302e020100300506032b656e04220420c8838e76d057dfb7d8c95a69e138160ad' +
+           'd6373fd71a4d276bb56e3a81b64ff61',
+    spki: '302a300506032b656e0321001cf2b1e6022ec537371ed7f53e54fa1154d83e98eb' +
+          '64ea51fae5b3307cfe9706',
+    result: '2768409dfab99ec23b8c89b93ff5880295f76176088f89e43dfebe7ea1950008'
+  },
+  {
+    name: 'X448',
+    size: 56,
+    pkcs8: '3046020100300506032b656f043a043858c7d29a3eb519b29d00cfb191bb64fc6' +
+           'd8a42d8f17176272b89f2272d1819295c6525c0829671b052ef0727530f188e31' +
+           'd0cc53bf26929e',
+    spki: '3042300506032b656f033900b604a1d1a5cd1d9426d561ef630a9eb16cbe69d5b9' +
+          'ca615edc53633efb52ea31e6e6a0a1dbacc6e76cbce6482d7e4ba3d55d9e802765' +
+          'ce6f',
+    result: 'f0f6c5f17f94f4291eab7178866d37ec8906dd6c514143dc85be7cf28deff39b' +
+            '726e0f6dcf810eb594dca97b4882bd44c43ea7dc67f49a4e',
```

Comment:
Same question for these, did you generate these or is there some reference?

Reply:
same answer